### ISSUE TITLE
Display wrong deposit value

### ISSUE THREAD

**Issue by ckypper** [antal-k/csgoempire-deposit-bot#21]
https://github.com/antal-k/csgoempire-deposit-bot/blob/77c15b0fd7bb9733774a7fc6c89111900529d9ad/src/services/csgoempire.ts#L183

This show the price when user deposit it, it'll be wrong when someone bid the item, it is not the final price

**Comment by antal-k** [comment]
Thanks! I'll take a look.

**Comment by antal-k** [comment]
Socket event:

`42 / trade, ["trade_status", [{
    "type": "deposit",
    "data": {
        "item": {
            "market_name": "Desert Eagle | Trigger Discipline (Field-Tested)",
            "market_value": 0.49,
            "asset_id": 27783698678
        },
        "status": -1,
        "status_message": "Auction withdrawal internal API call failed - Failed due to internal issue - You don't have enough space in your CS:GO inventory to complete this withdrawal.",
        "id": 99832056,
        "tradeoffer_id": 0
    }
}]]`

Basically, no chance of getting it directly right now. Maybe with some workaround.

**Comment by ckypper** [comment]
You can get it on "Confirming" status on total_value field but only when someone bid your item, if it's not bidding the total_value field on "Confirming" status will be undefined

**Comment by antal-k** [comment]
a542c431e713d071c222097a990e7a76f8d94a1f
